include/group.php

```diff
@@ -144,12 +140,7 @@ function group_add_member($uid,$name,$member,$gid = 0) {
 				// we indicate success because the group member was in fact created
 				// -- It was just created at another time
  	if (! dbm::is_result($r)) {
-		$r = q("INSERT INTO `group_member` (`uid`, `gid`, `contact-id`)
-			VALUES( %d, %d, %d ) ",
-			intval($uid),
-			intval($gid),
-			intval($member)
```

Comment:
Jepp!

And for the English speaking community: There is no need to do escaping with prepared statements - which are used for the new database functions.